nautobot/extras/filters.py

```diff
@@ -327,6 +327,15 @@ def __init__(self, custom_field, *args, **kwargs):
             # Contains handles lists within the JSON data for multi select fields
             self.lookup_expr = "contains"
 
+    def filter(self, qs, value):
+        NULL_FIELDS = ("null", "0")
```

Comment:
Does this make it so that we can't query for values that actually == 0? If so I don't think I'm in favor of that.

Reply:
I could use only null